src/aznfswatchdog

```diff
@@ -131,13 +131,26 @@ reconcile_conntrack()
 
 vecho "Starting aznfswatchdog..."
 
+# Detect and log distro and bash version
+if [ -f /etc/centos-release ]; then
+    linux_distro=$(cat /etc/centos-release)
+elif [ -f /etc/os-release ]; then
+    linux_distro=$(grep "^PRETTY_NAME=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
```

Comment:
how does the values look for "Pretty_Name" in this file? I used "ID" to get name of the distro before. Please verify this for all distro.

Reply:
Pretty_Name is I think more complete data point, it's combined of Name of distro + version.

for example:

NAME="Rocky Linux"
VERSION="8.7 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.7"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.7 (Green Obsidian)"